extensions/oidc/runtime/pom.xml

```diff
@@ -30,6 +30,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-security</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jsonp</artifactId>
+        </dependency>
```

Comment:
I don't know if the right fix is to add this one or to depend on `quarkus-smallrye-jwt`. I let you judge of that, not knowing exactly what the SmallRye JWT extension is doing (but my fear is that we might miss some important native image-related stuff somehow).

In any case, you also need to add the `-deployment` dependency to the deployment module. That's why the tests are failing right now.

Reply:
@gsmet sorry, I was working in replacing the test code with the Jackson provider and it worked, but not sure why I forgot to recompile all of the `oidc` extension...Let me answer about `quarkus-smallrye-jwt` in a few mins